packages/block/src/header-from-rpc.ts

```diff
@@ -26,6 +26,7 @@ export default function blockHeaderFromRpc(blockParams: any, options?: BlockOpti
     extraData,
     mixHash,
     nonce,
+    baseFeePerGas
```

Comment:
Hi there, thanks for the PR! 😄 

Hmm, this is then not working with pre-London blocks any more, so I guess we need a more differentiated solution here?

I just tested the following to be sure:

![grafik](https://user-images.githubusercontent.com/931137/124085413-4e402e80-da50-11eb-8e67-73812fe858f9.png)

Reply:
addressed!

Reply:
I would think this would be okay since `blockParams` is typed as `any`?